akka-persistence/src/main/scala/akka/persistence/journal/leveldb/LeveldbJournal.scala

```diff
@@ -20,9 +20,15 @@ import com.typesafe.config.Config
  *
  * Journal backed by a local LevelDB store. For production use.
  */
-private[persistence] class LeveldbJournal(val config: Config) extends AsyncWriteJournal with LeveldbStore {
+private[persistence] class LeveldbJournal(cfg: Config) extends AsyncWriteJournal with LeveldbStore {
   import LeveldbJournal._
 
+  def this() = this(null)
```

Comment:
I don't like the null, someone could accidentally use it.
I propose to instead do:

```scala
def this() = this(context.system.settings.config.getConfig("akka.persistence.journal.leveldb"))
```